tests/phpunit/tests/block-template-utils.php

```diff
@@ -382,4 +384,84 @@ public function test_wp_generate_block_templates_export_file() {
 		}
 		$this->assertTrue( $has_html_files, 'contains at least one html file' );
 	}
+
+	/**
+	 * Should run `do_shortcode` in block template part.
+	 *
+	 * @ticket 56780
+	 * @covers ::block_template_part
```

Comment:
```suggestion
	 * @ticket 56780
	 *
	 * @covers ::block_template_part
```

Per [56070](https://core.trac.wordpress.org/ticket/56070) proposal, There should be a line break between them.

Similar applies to other tests.

Reply:
@mukeshpanchal27 I've updated the DocBlock formatting for the tests.